### PR TITLE
plc ex.7.3

### LINKED ISSUE #57: PLC 7.3

### PR DESCRIPTION
**Pull request by Arneproductions**
fixed #57

### CODE REVIEW

File: Assignment-06/README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ Run `compile.sh` in the docker container
 
 `cleanup.sh` is provided to remove all files created by the compilation process.
 
+## PLC-ex4.3
```
Comment (amalieboegild) [inline Assignment-06/README.md:7]: You mean 7.3, right?

**Review by amalieboegild — CHANGES_REQUESTED** [review]

**Review by albe2669 — CHANGES_REQUESTED** [review]
Without knowing, is all the files you have included necessary? I like what we've been doing where we just say what files we need, that doesn't actually need changing. Also, i think there may be some compiled files left over no?

File: Assignment-06/README.md (hunk 1 above)
Comment (Arneproductions) [reply Assignment-06/README.md:7]: Yes

**Comment by Arneproductions** [comment]
> Without knowing, is all the files you have included necessary? I like what we've been doing where we just say what files we need, that doesn't actually need changing. Also, i think there may be some compiled files left over no?

Yes I do too... All the files in 7.1 and 7.2 will be removed when the branch created by @amalieboegild has been merged.

**Review by amalieboegild — APPROVED** [review]

**Review by albe2669 — APPROVED** [review]
Read thoroughly and well. Looks lovely

**Pull request merged**